scipy/spatial/kdtree.py

```diff
@@ -695,6 +695,70 @@ def traverse_no_checking(node1, node2):
                           other.tree, Rectangle(other.maxes, other.mins))
         return results
 
+    def __query_range(self,rng):
```

Comment:
Let's don't call the argument `rng`. The abbr is more often reserved for random number generators. perhaps query_range(self, min, max) where min and max are both broadcastable to (:, self.m) arrays.

Reply:
hmm ok, i can make that change. I don't know if it's as intuitive to think about the query hyper rectangle being defined over 2 different arguments but I suppose it does eliminate the possibility of accidental interleaved input ([min_i,max_i,...min_n,max_n])... if anyone else has feelings one way or another feel free to weigh in, other wise i'll make @rainwoodman 's suggested change

Reply:
oh that is spiffy! cool, i'll make the change shortly.